docs/content/developer/tutorials/sponsored-transactions/gas-station-server.mdx

```diff
@@ -1,11 +1,21 @@
 # The Gas Station Server
 
-:::info IOTA Gas Station
-
+:::info
 This guide is for developers who want to build their own gas station. If you'd rather just use sponsored transactions, IOTA provides a [Gas Station Component](../../../operator/gas-station/gas-station.mdx) that you can deploy in your environment.
-
 :::
```

Comment:
Maybe we should put some more emphasis on using the gas station? Like in most cases you should be fine with using the GS. So maybe we recommend checking out if the GS station fits your use case? I don't know, someone can probably come up with a better sentence 😅

Reply:
I tried to enforce that in the landing page of the tutorial:
https://github.com/iotaledger/iota/blob/devx/add-gas-station-usage/docs/content/developer/tutorials/sponsored-transactions/sponsored-txs.mdx#development-options
Wouldn't that be enough?